MapleServer2/Packets/MailPacket.cs

```diff
@@ -0,0 +1,141 @@
+using System;
+using System.Collections.Generic;
+using MaplePacketLib2.Tools;
+using MapleServer2.Constants;
+using MapleServer2.Types;
+using MapleServer2.Servers.Game;
+using MapleServer2.Data;
+using MapleServer2.Packets.Helpers;
+using MapleServer2.Tools;
+
+namespace MapleServer2.Packets
+{
+    public static class MailPacket
+    {
+        public static Packet Notify(GameSession session)
+        {
+            PacketWriter pWriter = PacketWriter.Of(SendOp.MAIL);
+
+            pWriter.WriteByte(14); // Mode for mail notification
+            pWriter.WriteInt(session.Mailbox.GetUnread()); // Count of unread mail
+            pWriter.WriteByte(); // Unknown
+            pWriter.WriteInt(); // Unknown maybe repeat of count?
+
+            return pWriter;
+        }
+
+        public static void Open(GameSession session)
```

Comment:
Same as before, I am not sure we should be having methods inside a packet class sending multiple packets